packages/auth/src/grant/routes.ts

```diff
@@ -206,13 +266,20 @@ async function continueGrant(
     interactionService
   } = deps
 
-  const interaction = await interactionService.getByRef(interactRef)
+  if (!ctx.request.body || Object.keys(ctx.request.body).length === 0) {
+    await pollGrantContinuation(deps, ctx, continueId, continueToken)
+    return
+  }
+
+  const interaction = await interactionService.getByRef(interactRef as string)
```

Comment:
Looks like `interactRef` could be `undefined` since its now optional. Could check it before line 274 instead of the string type cast to preserve the same error response as before for a required and missing `interactRef`, as well as making it more typesafe.

```ts
  if (!interactRef) {
    ctx.throw(401, { error: 'invalid_request' })
  }
 ```
 
Might also be able to keep [this 'Cannot issue access token without continue token' test](https://github.com/interledger/rafiki/pull/2285/files#diff-a9f1841aeba665aeaf6b6a3e2065cc618245ab8c7a5124641ecd655b24a4cd0dR508) and adapt to this case.  `interactRef required if no continueId`, or similar.